src/components/slider/slider.js

```diff
@@ -365,9 +488,7 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
      * @param x
      */
     function doSlide( x ) {
-      scope.$evalAsync( function() {
-        setModelValue( percentToValue( positionToPercent(x) ));
-      });
+      setModelValue( percentToValue( positionToPercent(x) ));
```

Comment:
I am always weary to remove existing code; especially a `$evalAsync()`. Did this fix a bug/issue you were having?